site/pages/resources/index.tsx

```diff
@@ -6,10 +6,31 @@ import { GetStaticPropsContext } from "next";
 export async function getStaticProps(ctx: GetStaticPropsContext) {
   try {
     const { locale } = ctx;
-    const documents = await fetchCMSContent("allDocuments", { locale });
-    const featuredArticles = await fetchCMSContent("allFeaturedPosts", {
-      locale,
-    });
+
+    const [
+      enDocuments,
+      enFeaturedArticles,
+      localeDocuments,
+      localeFeaturedArticles,
+    ] = await Promise.all([
+      fetchCMSContent("allDocuments", { locale: "en" }),
+      fetchCMSContent("allFeaturedPosts", { locale: "en" }),
+      fetchCMSContent("allDocuments", { locale }),
+      fetchCMSContent("allFeaturedPosts", { locale }),
+    ]);
+
+    let documents, featuredArticles;
+    // if en has more documents or articles than locale, then show locale documents and articles plus en documents and articles
+    if (
+      localeDocuments.length < enDocuments.length ||
+      localeFeaturedArticles.length < enFeaturedArticles.length
```

Comment:
```suggestion
      localeDocuments.length < enDocuments.length ||
      localeFeaturedArticles.length < enFeaturedArticles.length
```

Won't this always be true? Unless the local is `en`, in which case we could have skipped the `locale` queries anyways.

Reply:
The idea was if there were an equal number of locale and en then there were no omissions and just to show the locale language. But sounds like this is moot point given the below feedback (using groq vs handling in getStaticProps)